frontend2/src/utils/event.ts

```diff
@@ -0,0 +1,3 @@
+export function preventEventDefault(event: Event) {
```

Comment:
Updated

Reply:
So we can, but it'll make some code a bit uglier. We have two places that do this:

```
onEscapeKeyDown={dismissible ? undefined : preventEventDefault}
onPointerDownOutside={dismissible ? undefined : preventEventDefault}
onInteractOutside={dismissible ? undefined : preventEventDefault}
```

If we inline the function it will become this after formatting:

```
onEscapeKeyDown={
  dismissible
    ? undefined
    : (e) => {
        e.preventDefault();
      }
}
onPointerDownOutside={
  dismissible
    ? undefined
    : (e) => {
        e.preventDefault();
      }
}
onInteractOutside={
  dismissible
    ? undefined
    : (e) => {
        e.preventDefault();
      }
}
```

Additionally, it'll be instantiating 3 functions on each render rather than just referencing a single static function.